include/constants/opponents.h

```diff
@@ -863,5 +865,7 @@
 
 #define TRAINERS_COUNT                      855
 #define MAX_TRAINERS_COUNT                  864
+#define TRAINER_PARTNER(partner)           (865 + partner)
```

Comment:
```suggestion
#define TRAINER_PARTNER(partner)           ((MAX_TRAINER_COUNT) + partner)
```
Or something like this, perhaps? I'm not sure why it needs to be one higher than `MAX_TRAINERS_COUNT` anyway, since that value is never reached

Reply:
What I intended was to use Steven's partner id and then any new partner use ids after